lib/elixir_internal_certificate/scraper/schemas/search_result.ex

```diff
@@ -9,13 +13,24 @@ defmodule ElixirInternalCertificate.Scraper.Schemas.SearchResult do
     field :html_response, :string
 
     belongs_to :user_search,
-               ElixirInternalCertificate.Scraper.Schemas.UserSearch,
+               UserSearch,
                foreign_key: :user_search_id
 
     has_many :url_results,
-             ElixirInternalCertificate.Scraper.Schemas.UrlResult,
+             UrlResult,
              foreign_key: :search_result_id
 
     timestamps()
   end
+
+  def search_result_changeset(search_result \\ %__MODULE__{}, attrs),
+    do:
+      change(search_result, %{
```

Comment:
instead of using the change method like this, should we use this solution

```
search_result
|> cast(attrs, [fields])
|> validate_required([xxx]}
```

example 

![image](https://user-images.githubusercontent.com/11751745/176083893-58c5d89c-37ad-4775-8862-d063cfc91cc9.png)


The pros is with that solution, if the field is empty, or invalid, we will have the error, while with the `change` here, it doesn't return any error.

Reply:
Fixed in dbda9b9